e2e/tests/test_trtllm_checkpoint.py

```
@@ -0,0 +1,301 @@
+            run.snapshot_name,
+            run.source_pod,
+            source.metadata.uid,
+        )
+        pod_snapshot, content = snap.wait_for_snapshot_ready(
+            config.namespace,
+            run.snapshot_name,
+            timeout=1800,
+        )
+
+        ready = snap.condition(pod_snapshot, "Ready")
+        assert ready and ready.get("status") == "True"
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)